components/TradeForm.tsx

```diff
@@ -111,6 +168,30 @@ export default function TradeForm() {
       s.tradeForm.tradeType = type
     })
 
+  const leverageQuoteCalc = (leveragePct) => {
+    //get limit price or use markprice
+    const usePrice = Number(price) || markPrice
+    //calc current & available margin
+    const maxMarginUSD = accountEquity * 5
```

Comment:
i am not 100% sure that equity * 5 is the best way here, you can use getAssetsVal to get the value of all open orders + deposits + fee rebates accrued